wiki/markdown/statusCodes.md

```diff
@@ -170,6 +170,7 @@ These response can be generated only when `/generateAndPublish` endpoint is used
 | 401         | Unauthorized          | Unauthorized. Please, check if LDAP for REMReM Generate Service is disabled | Is returned if LDAP for REMReM Generate Service is enabled and REMReM Generate Publish have not access to it.          |
 | 406         | Not Acceptable        | No event id found with ERLookup properties                                  | Is returned if no event id fetched from configured event repository in REMReM generate.                                |
 | 417         | Expectation Failed    | Multiple event ids found with ERLookup properties                           | Is returned if multiple event ids fetched from configured event repository in REMReM generate.                         |
+| 422         | Unprocessable Entity  | Link specific options could not fetch information from ER                   | Is returned if could not fetch information from ER with specific options.                                              |
```

Comment:
See my comment in the review of remrem generate

Reply:
changed error message as suggested